README.md

```diff
@@ -219,7 +219,9 @@ jobs:
       object_prefix: ubuntu-repos
 ```
 
-### File selection: By name
+### File selection
+
+#### By name
```

Comment:
When you add/change headings in this file, you also need to adjust the TOC at the top.

Reply:
The entry in the TOC already conforms to the new heading.